Exec/science/wdconvect/GNUmakefile

```diff
@@ -14,6 +14,8 @@ MAESTROEX_HOME  := ../../..
 # Set the EOS, conductivity, and network directories
 # We first check if these exist in $(MAESTROEX_HOME)/Microphysics/(EOS/conductivity/networks)
 # If not we use the version in $(MICROPHYSICS_HOME)/Microphysics/(EOS/conductivity/networks)
+
+USE_FORT_MICROPHYSICS := TRUE
```

Comment:
Should this not be FALSE?

Reply:
no.  `ignition_chamulak` is the one net that is still in Fortran,  So for this problem only we need to build with Fortran.